modyn/supervisor/internal/pipeline_executor/pipeline_executor.py

```diff
@@ -9,15 +9,21 @@
 from typing import Any, Generator, Optional
 
 from modyn.common.benchmark import Stopwatch
-from modyn.supervisor.internal.evaluation_result_writer import LogResultWriter
+
+# pylint: disable-next=no-name-in-module
+from modyn.evaluator.internal.grpc.generated.evaluator_pb2 import EvaluateModelResponse, EvaluationAbortedReason
+from modyn.supervisor.internal.eval_strategies.abstract_eval_strategy import AbstractEvalStrategy
+from modyn.supervisor.internal.evaluation_result_writer import JsonResultWriter
 from modyn.supervisor.internal.grpc.enums import CounterAction, IdType, MsgType, PipelineStage
 from modyn.supervisor.internal.grpc.template_msg import counter_submsg, dataset_submsg, id_submsg, pipeline_stage_msg
 from modyn.supervisor.internal.grpc_handler import GRPCHandler
 from modyn.supervisor.internal.triggers import Trigger
+from modyn.supervisor.internal.utils import EvaluationStatusReporter
 from modyn.utils import dynamic_module_import
 
 logger = logging.getLogger(__name__)
 EXCEPTION_EXITCODE = 8
+EVALUATION_RESULTS = "evaluation_results"
```

Comment:
we need a different name for this global. Is this a directory name? Something else? Some suffix might be needed

Reply:
ok I see it's a key - maybe `EVAL_RESULTS_KEY` or `EVAL_RESULTS_IDENTIFIER` or something

Reply:
I think still `EVALUATION_RESULTS` is better because under this entry, the sub-object includes everything about evaluations -- different intervals, different metrics, different eval datasets (I will restore the setting of multi eval datasets).